Paws4Me/Paws4Me/Controllers/Logic/SignInViewController.swift

```diff
@@ -56,16 +65,3 @@ func addErrorBorderBoth(_ textField1: UITextField, _ textField2: UITextField,
     textField1.layer.addSublayer(bottomLine)
```

Comment:
Can this be used as a UITextField extension?

Reply:
I agree with this. I am not sure why I missed this.